media-video/obs-studio/obs-studio-9999.ebuild

```diff
@@ -216,13 +243,13 @@ src_configure() {
 			-DENABLE_SCRIPTING=ON
 		)
 	else
-		mycmakeargs+=( -DENABLE_SCRIPTING=OFF )
+		mycmakeargs+=(-DENABLE_SCRIPTING=OFF)
```

Comment:
isn't the += supposed to maintain the spaces?

Reply:
It's just style, the expansion happens later.

Reply:
but if you what i can revert to the old style

Reply:
> but if you what i can revert to the old style

Nono don’t worry. IIRC some years ago was an error but if Sam says it’s ok I’m more than ok with it